#goal definition
uint32 Height #lift 
uint32 isUp # angle gripper 
uint32 isClosed 
uint32 isUnload

---
#result definition
bool Ok
---
#feedback
string Status